app/view/grid/ItemDeleter.js

```diff
@@ -0,0 +1,47 @@
+Ext.define('CpsiMapview.view.grid.ItemDeleter',
+    {
+        extend: 'Ext.grid.column.Action',
+        xtype: 'cmv_itemdeleter',
+        width: 55,
+        requiredRole: this.deleteItemRequiredRole,
+        menuDisabled: true, // disable the column header menu containing sort/hide options
+        iconCls: 'x-fa fa-delete',
+        ignoreConfirmation: false,
+        isActionColumn: true,
+        hideable: false, // prevent the user from hiding this column
+        draggable: false,
+        tooltip: 'Click here to remove the record',
+
+        /**
+            Possible bug when a row editor plugin is added to the same grid as the ItemDeleter?
```

Comment:
Can probably remove this comment now?

Reply:
I'd leave it there actually because I didn't have a chance of testing whether that's still the case or not, and if that still is and should it create issues we'd spand time to maybe get to the same result, so this comment might save us a lot of time? Or maybe not? Should I remove it anyway?

Reply:
I think always having the editor there will ensure the bug won't happen. These comments should be found when looking at the pull request so think it is ok to remove now. Not seen this bug in 5 years!